parser/dirs.go

```diff
@@ -62,7 +63,7 @@ func walkDir(dir, rel string, walkFn walkFunc) error {
 }
 
 // parseDir is like go/parser.ParseDir but it constructs *est.File objects instead.
-func parseDir(fset *token.FileSet, dir, relPath string, filter func(os.FileInfo) bool, mode goparser.Mode) (pkgs map[string]*ast.Package, files []*est.File, err error) {
+func parseDir(context build.Context, fset *token.FileSet, dir, relPath string, filter func(os.FileInfo) bool, mode goparser.Mode) (pkgs map[string]*ast.Package, files []*est.File, err error) {
```

Comment:
Nit; maybe rename this to `buildContext` or something; at a glance I thought this was `context.Context`.